index.ts

```diff
@@ -27,6 +27,8 @@ export interface PluginConfig {
      * @default './src'
      */
     srcDir?: string | undefined
+
+    react?: boolean
```

Comment:
Do you think we could detect the presence of the react plugin in one of the hooks instead of requiring an extra config option?

Reply:
Finally had a chance to look at this, and it turns out - yes! So the latest version removes the "react" configuration option and automatically loads the preamble code from the React plugin if it sees it in the plugin list.